packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testRevertingToNonExistingVersion.st

```diff
@@ -0,0 +1,8 @@
+as yet unclassified
+testRevertingToNonExistingVersion
+	| instance check |
+	check := true.
+	self mock: #savedVersion of: SClassDiagram  toReturn: nil  during: [
+	instance := klass new.
+	[instance revert on: Error do: [ check := false]].
```

Comment:
What's the meaning of the`]` in this case it should do some type of exception handling?